rs/config/src/embedders.rs

```diff
@@ -152,24 +110,27 @@ pub struct FeatureFlags {
     pub write_barrier: FlagStatus,
     /// Indicates whether the support for 64 bit main memory is enabled
     pub wasm64: FlagStatus,
-    /// Rollout stage of the best-effort responses feature.
-    pub best_effort_responses: BestEffortResponsesFeature,
     /// Collect a backtrace from the canister when it panics.
     pub canister_backtrace: FlagStatus,
 }
 
-impl Default for FeatureFlags {
-    fn default() -> Self {
+impl FeatureFlags {
+    const fn const_default() -> Self {
         Self {
             rate_limiting_of_debug_prints: FlagStatus::Enabled,
             write_barrier: FlagStatus::Disabled,
             wasm64: FlagStatus::Enabled,
-            best_effort_responses: BestEffortResponsesFeature::ApplicationSubnetsOnly,
```

Comment:
If I understand correctly we are completely dropping the feature flag here, meaning we'd give up our ability to quickly disable this feature. If so, I think we should wait with merging this until the feature has been running on all subnets for a while.